### PR TITLE
Java 11.3: Varargs and Ambiguity #2219

### LINKED ISSUE #2219: Java 11.3: Varargs and Ambiguity
### Description

Add audio, documentation, and video regarding the **Varargs and Ambiguity**.

____

### Location of File to be added

The files should be added inside the `Varargs and Ambiguity`  folder which is inside the `Varargs`  folder present in the   `Java`  main folder.  The documentation file can be in the `.md`/`.py`/`.ipynb` format. Audio should be in `.mp3` format and to be submitted through a Google drive link. Video can be in any format and should be submitted through a Google Drive link.

 **Note:**
 - You are required to comment with `\assign CONTENT_TYPE`, so that other content types can be created at the same time.
 - Failing to comment in the above manner will lead to the removal of assignment from the issue even if the bot assigns.
 - You are required to do a PR within 7 days otherwise you will be unassigned and others will be assigned.
 - Issues will be assigned on a first come first serve basis.

### Domain

Java_Domain

### Type of Contribution

Audio, Video, Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by ishivamgupta9**
<hr>

## Description 📜

I have created documentation on  Java 11.3: Varargs and Ambiguity.
Fixes #2219  

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Java_Domain

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸
Screenshot (598)

<hr>

### CODE REVIEW

**Review by sumitaccess007 — CHANGES_REQUESTED** [review]
@ishivamgupta9,

Please help to know below points -
1. Under Issue #2219, with PR #2914, Why are you submitting changes made on content related to Issue - #1106 
Here in this PR, you have also modified file 'Impact_of_Java_on_Internet.md', which is not part of this issue - 'Varargs and Ambiguity'.
Please clarify.

Please make sure that you only submit content related to particular topic on which Issue is created.

File: Java/Varargs/Varargs_And_Ambiguity/Varargs_And_Ambiguity.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+```        
+ So here instead of overloading, we can store in an array as shown above and then traverse and add the result.
+
+  
+### Ambiguity
+
+A Java class can implement multiple interfaces and each interface can have some variables with respective name.
+Now, suppose a class implements two interfaces but both have one or more variables of same type and same variable name.
+Now if the interface variable is us the class implementing the interfaces are confused which . This is the ambiguity problem in java.
```
Comment (sumitaccess007) [inline Java/Varargs/Varargs_And_Ambiguity/Varargs_And_Ambiguity.md:92]: @ishivamgupta9 

Also in next commit, improve this line as well.

**Review by sumitaccess007 — CHANGES_REQUESTED** [review]

**Comment by ishivamgupta9** [comment]
@sumitaccess007 I had not changed that other issue, only for looking for the reference I had opened that file but not changed anything you can check it with original content.

**Comment by sumitaccess007** [comment]
@ishivamgupta9 - Verified it with original content. Below are the changes which are also part of this PR.

1. Number of files changed are two.
image

2. Original Content -
image

3. Content in Your Repo - 
It is fine now, the lines are deleted.

You can submit the PR now.

**Pull request closed without merging**